src/main/java/com/eternalcode/check/configuration/composer/StringComposer.java

```diff
@@ -0,0 +1,21 @@
+package com.eternalcode.check.configuration.composer;
+
+import panda.std.Result;
+
+public class StringComposer implements SimpleComposer<String>{
+
+    @Override
+    public Result<String, Exception> deserialize(String source) {
+        if (!source.startsWith("\"") || !source.endsWith("\"") || source.length() < 2) {
+            return Result.error(new IllegalStateException("Brakuje \" w confingu!"));
```

Comment:
Trzeba będzie dorobić jakąś fajną wiadomość do sprawdzania, jakieś kolorki do konsoli
coś w stylu Incorrect Syntax check the configuration on this website www.syntaxcheck.edu
(Dla przykładu
)